scripts/validate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Can we integrate this script into the CI?

Reply:
I would say we must integrate this with the current CI setup.

Anything else would not make sense to me.